docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2010 @@
+{
```

Comment:
<p>Let's use <code>model.compile_logp()(point)</code>?</p><p> </p><p><code>point_logps</code> is more of an internal specialized helper. <code>compile_logp</code> is more likely to be what users need and is more powerful.</p><p> </p><p>And, perhaps,  before even using <code>compile_logp</code> we could call <code>model.logpt</code>,print the graph and emphasize this graph represents the model logpt but like before it does not do any computations until you compile and pass in the inputs.  Then we coul use <code>.eval</code> for debugging, and then say one should compile it into a function for use in production, and that pymc model offers the helper <code>compile_logp</code> to make this eaiser.     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>

Reply:
+1